libvirt/tests/src/guest_os_booting/os_configuration/without_default_os_attributes.py

```
@@ -20,10 +75,16 @@ def run(test, params, env):
     3) Check the guest dumpxml.
     """
     vm_name = guest_os.get_vm(params)
-    firmware_type = params.get("firmware_type")
     os_dict = eval(params.get("os_dict"))
     host_arch = platform.machine()
-    os_xpath = eval(params.get("os_xpath"))
+    os_xpath = _adapt_os_xpath_uefi_disk_boot(
+        eval(params.get("os_xpath")), params, host_arch)
```

Ruff (0.15.12)

[error] Use of possibly insecure function; consider using `ast.literal_eval`

(S307)